drag/demo/demo.view.ts

```diff
@@ -39,7 +39,7 @@ namespace $.$$ {
 
 		transfer_adopt( transfer : DataTransfer ) {
 
-			const uri = transfer.getData( "text/uri-list" )
+			const uri = transfer.getData( "text/uri-list" ) == ""?transfer.getData( "text/plain" ):transfer.getData( "text/uri-list" )
```

Comment:
isn't better like this?
`const uri = transfer.getData( "text/uri-list" ) || transfer.getData( "text/plain" )`

Reply:
no, it's not

Reply:
let me try test and let you know.

Reply:
okay, it's better. Accept it

Reply:
I cannot commit here, please, insert on your own.

Reply:
https://github.com/hyoo-ru/mam_mol/pull/699/commits/2ef4a4cd48d9be2cb5df390f13f6d213c9172b32
done